code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -839,12 +839,19 @@
 	result_amount = 2
 
 /datum/chemical_reaction/space_cleaner
-	name = "Space cleaner"
+	name = "Space Cleaner"
 	id = CLEANER
 	result = CLEANER
 	required_reagents = list(AMMONIA = 1, WATER = 1)
 	result_amount = 2
 
+/datum/chemical_reaction/space_cleaner/bleach
```

Comment:
I wouldn't bother to change it if you're not already fixing other stuff. It literally won't affect anything, it was just something I wanted you to consider. For some things, it would matter if it inherited (for example, our vaporization reactions use inheritance in this file)